components/raftstore/src/store/fsm/peer.rs

```diff
@@ -4385,7 +4385,9 @@ where
         // do not clean up the cache, it may keep growing.
         let drop_cache_duration =
             self.ctx.cfg.raft_heartbeat_interval() + self.ctx.cfg.raft_entry_cache_life_time.0;
-        let cache_alive_limit = Instant::now() - drop_cache_duration;
+        let now = Instant::now();
+        let cache_alive_limit = now - drop_cache_duration;
```

Comment:
what is this cache_alive_limit for? Seems no code uses it.

Reply:
It's used at L4415.